README.md

```diff
@@ -13,10 +13,22 @@ Note that this project makes use of event emitting capabilities of [RTCPeerConne
 
 ## API
 
-`constructor(peerConnection, opts)`
-
-`collectStats()`
-
-`collectTraces()`
-
-`collectInitialConnectionStats`
+`constructor(peerConnection: RTCPeerConnection, opts: StatsGathererOpts)`
+
+```
+interface StatsGathererOpts {
+  session?: string;     // sessionId
+  initiator?: string;
+  conference?: string;  // conversationId
```

Comment:
What's the reasoning for naming this `conference` and not just `conversationId`?